kapitan/cli.py

```diff
@@ -361,14 +364,33 @@ def secret_update(args, ref_controller):
                 logger.warning("WARNING: parameters.kapitan.secrets.recipients is deprecated, " +
                     "please use parameters.kapitan.secrets.gpg.recipients")
                 recipients = inv['nodes'][args.target_name]['parameters']['kapitan']['secrets']['recipients']
+        if not recipients:
```

Comment:
Same here.
```
recipients = [dict((("name", name),)) for name in args.recipients]

if not recipients:
    raise KapitanError(..)
```

Reply:
location is correct, needs to be done after `args.target_name` check